src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -921,4 +970,6 @@ Now you can see all NFTs
 
 You are able to create an NFT collection marketplace from the `ligo/fa` library.
 
+On next training, you will add the Buy and Sell functions to your smart contract and update the frontend to allow these actions.
+
```

Comment:
My MintPage.tsx also doesn't look like the solution. We need to be clear where code goes, because I didn't put everything inside the `MintPage()` component function.

Reply:
ok, I saw your code. People not used to React would paste code at the beginning of the file and not inside the React component. 
Got it, I will try to be more precise on this for the next version (coming very soon)